test/over_react_redux/connect_test.dart

```diff
@@ -70,17 +70,17 @@ main() {
 
     test('throws when multiple `mapStateToProps` type args are provided', () {
       expect(() => connect<CounterState, CounterProps>(
-          mapStateToProps: (state){},
-          makeMapStateToProps: (state, ownProps){}
+          mapStateToProps: (state) => {},
+          makeMapStateToProps: (state, ownProps) => (_) => {}
```

Comment:
What exactly does this do? like what was the lint? this seems pointless. if this were JS sure it binds the `this` context but here it just means like `return`... was the lint like `missing-return-of-non-void-return-value-cause-im-dart`?

Reply:
Haha yep, exactly @kealjones-wk, just a missing return statement that should return a function